src/sage/modular/modform/element.py

```diff
@@ -2568,6 +2568,48 @@ def __mul__(self, other):
 
         return newparent.base_extend(newqexp.base_ring())(newqexp)
 
+    def _pow_int(self, n):
+        """
+        Raises ``self`` to integer powers.
+
+        TESTS::
+
+            sage: F = ModularForms(1, 12).0
+            sage: (F^5).qexp(20)
+            q^5 - 120*q^6 + 7020*q^7 - 266560*q^8 + 7379190*q^9 - 158562144*q^10 + 2748847640*q^11 -
+            39443189760*q^12 + 476711357265*q^13 - 4910778324400*q^14 + 43440479153652*q^15 -
+            331129448133120*q^16 + 2173189785854230*q^17 - 12199334429782080*q^18 +
+            57636170473930920*q^19 + O(q^20)
+
+            Exponentiation is a lot faster than chain multiplication:
+
+            sage: F = ModularForms(DirichletGroup(6).0, 3).0
+            sage: G = F**8                                                              # long time -- around 3 seconds
```

Comment:
What are you showcasing here? You are saying that exponentiation is a lot faster than chain multiplication, but only provide an example of exponentiation. By the way it is phrased, I was expecting an example of chained multiplication.

Anyhow, I not sure that this test is very pertinent since the `_pow_int` is hidden in the Sage reference manual. It would be more useful in the main documentation of the element class.

Reply:
Yeah, looking back I am not sure what this is testing :D I will remove it.